### PR TITLE
bugfix/status-correction

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## 🚨 Breaking Changes 🚨 (these are a result of the complementary source package changes)
- The default schema for the source tables are now built within a schema titled (`<target_schema>` + `_jira_source`) in your destination. The previous default schema was (`<target_schema>` + `_stg_jira`) for source. This may be overwritten if desired.
## 🎉 Documentation and Feature Updates
- Updated README documentation updates for easier navigation and setup of the dbt package
- Added `jira_[source_table_name]_identifier` variables to allow for easier flexibility of the package to refer to source tables with different names.
## Bug Fixes
- Corrected bug introduced in 0.8.0 that would prevent the correct `status` data from being passed to model `jira__daily_issue_field_history`.
  - Please note, a `dbt run --full-refresh` will be required after upgrading to this version in order to capture the updates.

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [X] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

Since this code change adjusts an incremental model. We will need this to be a breaking change.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue #60 
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🚉 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -55,9 +55,6 @@ seeds:
   jira_integration_tests:
     +column_types:
       _fivetran_synced: timestamp
-    board:
-      +column_types:
-        id: "{{ 'int64' if target.name == 'bigquery' else 'bigint' }}"
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:60]: This was removed as it was not used in the integration tests. 
image

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/jira_source
-  version: [">=0.4.0", "<0.5.0"]
\ No newline at end of file
+# - package: fivetran/jira_source
+#   version: [">=0.5.0", "<0.6.0"]
+  - git: https://github.com/fivetran/dbt_jira_source.git
+    revision: feature/standardization-updates
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: To be switched before merge

**Review by fivetran-catfritz — APPROVED** [review]
Reviewed changes, ran and tested locally--LGTM!

**Review by fivetran-reneeli — COMMENTED** [review]
Looks good!

**Pull request merged**